ffcx/ir/elementtables.py

```diff
@@ -341,7 +355,7 @@ def build_optimized_tables(
         # the dofmap offset may differ due to restriction.
 
         tdim = cell.topological_dimension()
```

Comment:
Could we add a comment here as to what happens below? I guess what is going on is that if the integral is an interior facet or mixed dimensional integral, we need to compute the "facet-permutations", to ensure that the codim-1 cell is aligned with the codim-0 facet?